YAKL_parallel_for_c.h

```diff
@@ -584,35 +584,41 @@ namespace c {
   }
 
 
-  template <class F> inline void parallel_for_cpu_serial( int ubnd , F const &f , int vectorSize = 128 ) {
+  template <class F> inline void parallel_for_cpu_serial( int ubnd , F const &f ) {
+    #pragma omp target teams distribute parallel for simd
```

Comment:
This needs to be wraped in an `#ifdef __OPENMP_45__` to make room for normal CPU openMP later. Same for all other instances.